content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -9,197 +9,81 @@ card:
-
-### Install a Hypervisor
-
-If you do not already have a hypervisor installed, install one of these now:
-
-• [HyperKit](https://github.com/moby/hyperkit)
-
-• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-• [VMware Fusion](https://www.vmware.com/products/fusion)
-
-### Install Minikube
-The easiest way to install Minikube on macOS is using [Homebrew](https://brew.sh):
+If the [Brew Package Manager](https://brew.sh/) installed:
 
 ```shell
 brew install minikube
 ```
 
-You can also install it on macOS by downloading a stand-alone binary:
-
-```shell
-curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
-  && chmod +x minikube
-```
-
-Here's an easy way to add the Minikube executable to your path:
+Otherwise, download minikube directly:
 
 ```shell
-sudo mv minikube /usr/local/bin
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64
+sudo install minikube-darwin-amd64 /usr/local/bin/minikube
 ```
 
 {{% /tab %}}
 {{% tab name="Windows" %}}
-### Install kubectl
-
-Make sure you have kubectl installed. You can install kubectl according to the instructions in [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
-
-### Install a Hypervisor
-
-If you do not already have a hypervisor installed, install one of these now:
-
-• [Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_install)
-
-• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-
-{{< note >}}
-Hyper-V can run on three versions of Windows 10: Windows 10 Enterprise, Windows 10 Professional, and Windows 10 Education.
-{{< /note >}}
 
-### Install Minikube using Chocolatey
-
-The easiest way to install Minikube on Windows is using [Chocolatey](https://chocolatey.org/) (run as an administrator):
+If the [Chocolatey Package Manager](https://chocolatey.org/) is installed, use it to install minikube:
 
 ```shell
 choco install minikube
 ```
 
-After Minikube has finished installing, close the current CLI session and restart. Minikube should have been added to your path automatically.
-
-### Install Minikube using an installer executable
-
-To install Minikube manually on Windows using [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), download [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) and execute the installer.
-
-### Install Minikube via direct download
-
-To install Minikube manually on Windows, download [`minikube-windows-amd64`](https://github.com/kubernetes/minikube/releases/latest), rename it to `minikube.exe`, and add it to your path.
+Otherwise, download and run the [Windows installer](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe)
```

Comment:
```suggestion
Otherwise, download and run the [Windows installer](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe).
```
(minor detail, OK to merge without heeding this)